integrationtests/org.eclipse.xtext.integrationtests.target/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.xtext</groupId>
+		<artifactId>org.eclipse.xtext.swtbot.parent</artifactId>
+		<version>2.17.0-SNAPSHOT</version>

```

Comment:
are we sure we want to keep this versioning scheme

Reply:
good point. these projects don't follow Xtext's versioning. Better start with 1.0.0-SNAPSHOT